addons/calendar/models/calendar_recurrence.py

```diff
@@ -334,7 +334,7 @@ def _stop_at(self, event):
     def _detach_events(self, events):
         events.write({
             'recurrence_id': False,
-            'recurrency': False,
+            'recurrency': True,
```

Comment:
I don't understand why we need this change. Could you explain?

Reply:
Got it! Thanks for the explanation. I'll just ask you to add some tests to make sure that what you implemented keeps working in the future and I'll green this :)

Reply:
Hello, @leonardopavanrocha I've implemented the changes.
Thank you :slightly_smiling_face: